mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/search/ElasticSearchSearcherTest.java

```diff
@@ -128,9 +128,9 @@ void tearDown() throws IOException {
     }
 
     @Test
-    void searchingInFiveMailboxesShouldReturnOnlyTheseMailboxesMessagesUid() throws Exception {
+    void searchingInALargeNumberOfMailboxesShouldReturnAllMailboxesMessagesUid() throws Exception {
         MailboxSession session = MailboxSessionUtil.create(USERNAME);
-        int numberOfMailboxes = 6;
+        int numberOfMailboxes = 700;
```

Comment:
What's the execution speed of this?

Reply:
14 seconds, anyway, it passes without the fix with a lower value

Reply:
Let s see if it is stable then